modules/social_features/social_user/src/Routing/RouteSubscriber.php

```diff
@@ -45,6 +45,21 @@ protected function alterRoutes(RouteCollection $collection) {
       $route->setOption('_admin_route', FALSE);
     }
 
+    // Routes for which needs to disable access if the user is blocked.
+    $disable_access_for = [
+      'view.groups.page_user_groups',
```

Comment:
@dmytriikaiun LU still have access to this route, because it has different _custom_access in `modules/social_features/social_group/src/Routing/RouteSubscriber.php`, so please check if we can fix it there.